BlueprintUICommonControls/Tests/Sources/AttributedLabelTests.swift

```diff
@@ -259,6 +259,95 @@ class AttributedLabelTests: XCTestCase {
         compareSnapshot(of: element)
     }
 
+    func test_linkTapDetection() {
```

Comment:
@watt Went ahead and added these tap detection snapshot tests

- [ ] Pull down actual correct snapshots from CI

Reply:
- [ ] Pull down actual correct snapshots from CI